spring-shell-docs/src/main/asciidoc/using-spring-shell.adoc

```diff
@@ -239,7 +239,7 @@ to specify the same parameter (so only one of them can be used). Here is an exam
 [source, java]
 ----
 	@ShellMethod("Describe a command.")
-	public String help(@ShellOption({"-C", "--command"} String command) {
```

Comment:
This is the signature of the actual `help` method, it was not meant for being copy/pasted and tried by the reader. I will clarify the preceding sentence in that sense!